metricbeat/module/zookeeper/mntr/_meta/fields.yml

```diff
@@ -5,6 +5,7 @@
     command.
   release: ga
   fields:
+    # ECS?
     - name: hostname
```

Comment:
Sounds like it

Reply:
follow up directly in PR https://github.com/elastic/beats/pull/10286